src/basis/data.js

```diff
@@ -2594,7 +2601,17 @@
       flushAllDataset();
     }
 
-    return function(state){
+    Dataset.preventAccumulations = function(dataset){
+      var entry = eventCache[dataset.basisObjectId];
+      if (entry && entry !== PREVENT_ACCUMULATIONS)
```

Comment:
Actually, no. It is useful, because omitting this may cause calling `realEvent` on `PREVENT_ACCUMULATIONS` what would give an error